### PR TITLE
This is my first commit

### PR DESCRIPTION
**Pull request by Passang001**

### CODE REVIEW

File: PassangWeek1/PassangWeek1/Program.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+﻿
+
+namespace MyFirstApp
```
Comment (kacharyea) [inline PassangWeek1/PassangWeek1/Program.cs]: Shouldn't the namespace be PasangWeek1?

File: PassangWeek1/PassangWeek1/Program.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
+﻿
+
```
Comment (kacharyea) [inline PassangWeek1/PassangWeek1/Program.cs]: Can you remove the whitespace line 1 and 2?

File: PassangWeek1/PassangWeek1/Program.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+namespace MyFirstApp
+{
+    internal class Program
```
Comment (kacharyea) [inline PassangWeek1/PassangWeek1/Program.cs]: Not a big deal, but I would make this class public.

**Review by kacharyea — CHANGES_REQUESTED** [review]
Can you resolve the comments?

File: PassangWeek1/PassangWeek1/Program.cs (hunk 1 above)
Comment (Passang001) [reply PassangWeek1/PassangWeek1/Program.cs]: done

**Review by kacharyea — APPROVED** [review]
Looks good!

**Pull request merged**